common/config/rush/yarn.lock

```diff
@@ -1494,7 +1494,7 @@
 
 "@rush-temp/create-fusion-app@file:./projects/create-fusion-app.tgz":
   version "0.0.0"
-  resolved "file:./projects/create-fusion-app.tgz#3e4b87baaa08effcbc6e2c73442a5e2aa34b33cc"
```

Comment:
What is this about?

Reply:
After inspecting a little more, when I run `rush update` it removes the hashes, but when I run `rush update --full --purge` it adds the hashes.

@lhorie @rtsao What is expected here? Is this safe to ignore?